package.json

```diff
@@ -68,12 +68,12 @@
 		"@types/vscode": "^1.66.0",
 		"@typescript-eslint/eslint-plugin": "^5.18.0",
 		"@typescript-eslint/parser": "^5.18.0",
+		"@vscode/test-electron": "^2.3.9",
 		"eslint": "^8.12.0",
 		"glob": "^7.2.0",
 		"mocha": "^9.2.2",
 		"sinon": "^17.0.1",
 		"text-encoding": "^0.7.0",
-		"typescript": "^4.6.3",
-		"vscode-test": "^1.6.1"
```

Comment:
[vscode-test](https://www.npmjs.com/package/vscode-test) is deprecated , replaced with [@vscode/test-electron](https://www.npmjs.com/package/@vscode/test-electron). Fixes issue on Windows.